xbmc/addons/AddonInstaller.h

```diff
@@ -90,6 +90,12 @@ class CAddonInstaller : public IJobCallback
    */
   bool HasJob(const std::string& ID) const;
 
+  /*! \brief Sets flag whether we're running 'Update all' from AddonBrowser
+   *  \param isUpdateAll true or false
+   */
+  void SetModeUpdateAll(bool isUpdateAll) { m_isUpdateAll = isUpdateAll; }
```

Comment:
Where is this used?

Reply:
encloses the loop when running "update all" from addonbrowser
https://github.com/xbmc/xbmc/blob/17a54f2c13694a775b2b278dcf998ff98223540e/xbmc/addons/GUIWindowAddonBrowser.cpp#L158-L163

needed to choose the correct function call:
https://github.com/xbmc/xbmc/blob/17a54f2c13694a775b2b278dcf998ff98223540e/xbmc/addons/AddonInstaller.cpp#L473-L482

otherwise we would run into issues fetching the wrong update (e.g. 1.0.3), happened during the last test

Reply:
this is obsolete now